src/index.css

```diff
@@ -1,7 +1,7 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+  font-family: 'Fira Sans', 'Roboto', -apple-system, BlinkMacSystemFont,
```

Comment:
I noticed the `fira sans` was is the font family list and moved that and Roboto,the Material UI default font, to the head of the list. This [tutorial](https://surajsharma.net/blog/react-material-ui-font-family) and [Material UI](https://material-ui.com/customization/typography/) walkthrough explain other ways to do this. I tried both and decided less code and fewer files were better.